src/pages/Sell/ListingPage.jsx

```diff
@@ -0,0 +1,96 @@
+import { useNavigate } from 'react-router-dom';
+import Input from '../../components/common/Input';
+import { useRef, useState } from 'react';
+import { useSellContext } from '../../contexts/SellContext';
+
+export default function ListingPage(_props) {
+  const [isLoading, setIsLoading] = useState(false);
+  const { setListing } = useSellContext();
+  const formRef = useRef(null);
+  const navigate = useNavigate();
+  async function handleSubmit(e) {
+    e.preventDefault();
+    setIsLoading(true);
+
+    if (formRef?.current) {
+      const formData = new FormData(formRef.current);
+      const data = {};
+      formData.forEach((value, key) => {
+        data[key] = value;
+      });
+
+      console.log(data);
```

Comment:
Did you run ESLint? I thought console.log should throw an error for console.log

Reply:
i did not lol, i can remove

Reply:
will do this and other lint/formatting stuff in another PR